book/chapters/chapter12/model_interpretation.qmd

```diff
@@ -150,16 +159,25 @@ effect = FeatureEffect$new(predictor, feature = "amount",
+@fig-iml-pdice shows that if the `amount` is smaller than roughly 10,000 then on average there is a high chance that the predicted creditworthiness will be `good`. Furthermore, the ICE curves are more or less parallel lines, meaning that there do not seem to be strong interactions present where `amount` is involved.
 
 ### Surrogate Models
 
-Interpretable models such as decision trees or linear models can be used as `r index("surrogate models", "surrogate model")` to approximate or mimic a (often very complex) black-box model.
+Interpretable models such as decision trees or linear models can be used as `r index("surrogate models", "surrogate model")` to approximate or mimic an often very complex black-box model.
 Inspecting the surrogate model can provide insights into the behavior of a black-box model, for example by looking at the model coefficients in a linear regression or splits in a decision tree.
-
 We differentiate between local surrogate models, which approximate a model locally around a specific data point of interest, and global surrogate models which approximate the model across the entire input space (@Ribeiro2016lime; @Molnar2022).
+
 The features used to train a surrogate model are usually the same features used to train the black-box model or at least data with the same distribution to ensure a representative input space.
-However, the target used to train the surrogate is the predictions obtained from the black-box model.
+However, the target used to train the surrogate model is the predictions obtained from the black-box model, not the real outcome of the underlying data.
+Hence, conclusions drawn from the surrogate model are valid if the surrogate model approximates the black-box model very well (i.e., if the model fidelity is high). 
+It is therefore important to measure and report also the approximation error of the surrogate model.
+
+The data used to train the black-box model may be very complex or limited, making it challenging to directly train a well-performing interpretable model on that data. 
+Having trained a black-box model that captures the relationships in the original data very well, enables us to generate new labeled data in specific regions of the input space with which we can augment the original data.
+<!-- During this 'surrogate data generation' process the predictions of the black-box model serve as labels. -->
+The augmented data can then be used to train an interpretable model that captures and explains the relationships learned by the black-box model (in specific regions) or to identify flaws or unexpected behavior.
+<!-- Surrogate data generation allows for the benefits of both the accuracy of the black-box model and the interpretability of the resulting interpretable model. -->
+<!-- The black-box model acts as a powerful predictor, providing labels for the artificially generated data, while the interpretable model can offer a better understanding in the targeted regions of the input space. -->
```

Comment:
@mllg @giuseppec I'm sorry but I don't follow this at all, and I'm not entirely sure it's relevance to the chapter.
Why are we now talking about artifically generated data?

Reply:
because artficially generated data is used to train the surrogate?

Reply:
Okay I don't think that was said before or really here, how about:

> The data used to train the black-box model may be very complex or limited, making it challenging to directly train a well-performing interpretable model on that data. 
Instead, we can use the black-box model to generate new labeled data in specific regions of the input space with which we can augment the original data.
The augmented data can then be used to train an interpretable model that captures and explains the relationships learned by the black-box model (in specific regions) or to identify flaws or unexpected behavior.

Reply:
probably fine